### PR TITLE
Fix Readme

### PR DESCRIPTION
**Pull request by peterberkenbosch**
Update README with fixes on the integration documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ And then write a file inside your app root under `data/home/default.json`:
 Use the content inside an existing view, e.g. `app/views/spree/home/index.html.erb`:
 
 ```erb
-<% data = SolidusContent::Entry.data_for(:home, :default) %>
```
Comment (elia) [inline README.md:57]: I'll open an issue for implementing this, it's certainly an API I'd like to have in this extension

File: README.md [hunk 2]
```diff
@@ -76,21 +77,23 @@ Then, visit `/c/home/default` or even just `/c/home` (when the content slug is
...
-  get "legal", to: 'solidus_content#show', type: :info, id: :legal
+  get "privacy", to: 'spree/solidus_content#show', type: :info, id: :privacy
+  get "legal", to: 'spree/solidus_content#show', type: :info, id: :legal
 
-  # Will render app/views/spree/solidus_content/info.html.erb
-  get "blog/:id", to: 'solidus_content#show', type: :post
+  # Will render app/views/spree/solidus_content/post.html.erb
+  get "blog/:id", to: 'spree/solidus_content#show', type: :post
+
+  mount Spree::Core::Engine, at: '/'
```
Comment (elia) [inline README.md:96]: Any specific reason for preferring app routes over solidus routes? I'm all but against it, I just like to learn if there's something I'm not aware about this.

**Review by elia — COMMENTED** [review]
just a couple questions/comments, but overall 👍

File: README.md (hunk 2 above)
Comment (peterberkenbosch) [reply README.md:96]: This was the only way I could make this work, errored out on root address already present. I could not make the sample work in my Rails app around the solidus routes. Maybe Rails 6 added some constraints there?

**Review by elia — APPROVED** [review]

**Pull request merged**